api/src/utils/helpers.ts

```diff
@@ -0,0 +1,45 @@
+import { IPitch } from '../models/pitch';
+
+export interface SessionUser {
+  firstName: string;
+  lastName: string;
+  preferredName: string;
+  email: string;
+  phone: string;
+  gender: [string];
+  pronouns: [string];
+  dateJoined: Date;
+  masthead: boolean;
+  onboarding: string;
+  portfolio: string;
+  linkedIn: string;
+  twitter: string;
+  claimedPitches: [IPitch];
+  submittedPitches: [IPitch];
+  currentTeams: [string];
+  role: string;
+  races: [string];
+  interests: [string];
+}
+
+export const sessionizeUser = (user: any): SessionUser => ({
```

Comment:
Instead of using the `any` type, you can `import { IUser } from '../models/user'` and type user with `IUser`, which will get rid of the warning